jax/_src/api.py

```diff
@@ -1580,7 +1580,7 @@ def pmap(
   # TODO(yashkatariya): Move this out after shard_map is out of experimental and
   # in _src
```

Comment:
Can we address this TODO (here or following up)?

Reply:
@yashk2810 you bro

Reply:
yeah I can follow up!